### ISSUE TITLE
Tabs are not accessible using keyboard

### ISSUE THREAD

**Issue by ediamin** [a8cteam51/special-projects-blocks-monorepo#163]
The tab buttons are not focusable, which causes an accessibility issue. There is no `tabindex` attribute in the .tab elements. If I click any button, then `tabindex="-1"` is added to all other buttons. I'm using the v0.1.1.

I've used the block on this page: https://opportunityhub.co/next/. You can find it in the "Choose Your Pathway" section.

https://github.com/user-attachments/assets/31c53329-8639-4158-a0c1-34db2e5a4f30
